### PR TITLE
Initial database dump

### PR DESCRIPTION
**Pull request by lisovyk**

### CODE REVIEW

**Review by lhalam — APPROVED** [review]

File: moninag/database_dump/moninag_dump.sql [hunk 1]
```diff
@@ -0,0 +1,104 @@
+TRUNCATE auth_permission, auth_group_permissions, check_check, contact_contact, django_content_type, nagplugin_nagplugin, registration_customuser, server_server, service_service;
```
Comment (lhalam) [inline moninag/database_dump/moninag_dump.sql]: delete this file

File: moninag/database_dump/moninag_dump.sql [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	(25, 'Can add nag plugin', 9, 'add_nagplugin'),
+	(26, 'Can change nag plugin', 9, 'change_nagplugin'),
+	(27, 'Can delete nag plugin', 9, 'delete_nagplugin'),
+	(28, 'Can add contact', 10, 'add_contact'),
+	(29, 'Can change contact', 10, 'change_contact'),
+	(30, 'Can delete contact', 10, 'delete_contact');
+
+INSERT INTO check_check (id, name, run_freq, service_id, target_port, plugin_id, last_run, output, status, state) VALUES
+	(12, 'TCP checker', 60, 5, 80, 6, '2017-03-14 15:07:20.074326+02', 'PING WARNING - Packet loss = 0%, RTA = 124.02 ms|rta=124.017998ms;100.000000;500.000000;0.000000 pl=0%;20;60;0', 'WARNING', 't'),
+	(9,	'DNS checker', 300, 4, 80, 1, '2017-03-17 10:10:31.485749+02', 'DNS OK: 0.051 seconds response time. softserve.ua returns 188.226.190.17|time=0.051400s;;;0.000000','OK', 't'),
```
Comment (ghost) [inline moninag/database_dump/moninag_dump.sql]: extra spaces 📐 but its not the most important thing

File: moninag/database_dump/moninag_dump.sql [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+	(13, 'HTTP checker', 60, 5, 80, 3, '2017-03-14 15:11:24.746931+02', 'HTTP OK: HTTP/1.1 301 Moved Permanently - 490 bytes in 0.108 second response time |time=0.108433s;;;0.000000 size=490B;;;0','OK', 't'),
+	(20, 'Time difference', 900, 12, 37, 7, '2017-03-14 15:05:02.433722+02', 'HTTP OK: HTTP/1.1 301 Moved Permanently - 103 bytes in 0.291 second response time |time=0.291413s;;;0.000000 size=103B;;;0','OK', 't'),
+	(10, 'Pinger', 900, 4, 80, 4, '2017-03-14 14:31:12.686155+02', 'PING CRITICAL - Packet loss = 100%|rta=500.000000ms;100.000000;500.000000;0.000000 pl=100%;20;60;0','FAIL', 't'),
+	(15, 'Pinger', 60, 7, 80, 4, '2017-03-14 15:13:32.056138+02', 'HTTP OK: HTTP/1.1 301 Moved Permanently - 103 bytes in 0.296 second response time |time=0.296464s;;;0.000000 size=103B;;;0','OK', 't'),
+	(11, 'HTTP checker', 60, 4, 80, 3, '2017-03-14 15:13:33.060569+02', 'PING CRITICAL - Packet loss = 100%|rta=500.000000ms;100.000000;500.000000;0.000000 pl=100%;20;60;0','FAIL', 't');
+
+INSERT INTO contact_contact (id, first_name, second_name, email, is_active, activation_key, user_id) VALUES
+	(1, 'Jonh', 'Snow', 'humetsky+12@gmail.com', 'f', 'cce43c02f9928b2b2fde5884a573a6e4b15971e7', 5);
```
Comment (ghost) [inline moninag/database_dump/moninag_dump.sql]: is it the proper email address ❓

File: moninag/database_dump/moninag_dump.sql [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+INSERT INTO service_service (id, name, status, server_id, state) VALUES
+	(11, 'FTP watcher', 'WARNING', 9, 't'),
+	(4, 'SoftServe watcher', 'FAIL', 4, 't'),
+	(6, 'Ping me!', 'WARNING', 5, 't'),
+	(12, 'All you need is time!', 'OK', 10, 'f'),
+	(9, 'Discover HTTP', 'OK', 7, 't'),
+	(10, 'Gmail', 'FAIL', 8, 't'),
+	(7, 'Ping Attack', 'OK', 6, 't'),
+	(5, 'Github watcher', 'OK', 5, 't');
```
Comment (HenrikhB) [inline moninag/database_dump/moninag_dump.sql]: why state is `t` ?

File: moninag/database_dump/moninag_dump.sql (hunk 3 above)
Comment (lisovyk) [reply moninag/database_dump/moninag_dump.sql]: It was on demo 3, i think so

File: moninag/database_dump/moninag_dump.sql (hunk 2 above)
Comment (lisovyk) [reply moninag/database_dump/moninag_dump.sql]: it is an old file and will be deleted :smiling_imp:

File: moninag/database_dump/moninag_dump.sql (hunk 3 above)
Comment (HenrikhB) [inline moninag/database_dump/moninag_dump.sql]: on demo3 it was _**lv219py@gmail.com**_

**Pull request merged**